src/components/gallery/gallery.css

```diff
@@ -0,0 +1,57 @@
+
+
+* {
+    box-sizing: border-box;
+    margin:0;
+    padding:0;
+}
+
+body {
```

Comment:
This style block is problematic, it prevents the app filling the frame. Maybe this and the above * block are left in by mistake?

Reply:
My bad, I left it by mistake. Thank you!